runtimeconfig/tests/unit/test_config.py

```diff
@@ -145,14 +145,14 @@ def test_variable(self):
         self.assertEqual(len(conn._requested), 0)
 
     def test_get_variable_w_bound_client(self):
-        from google.cloud._helpers import _rfc3339_to_datetime
+        from google.api_core import datetime_helpers
 
         VARIABLE_NAME = 'my-variable/abcd'
         VARIABLE_PATH = '%s/variables/%s' % (self.CONFIG_PATH, VARIABLE_NAME)
         RESOURCE = {
             'name': VARIABLE_PATH,
             'value': 'bXktdmFyaWFibGUtdmFsdWU=',  # base64 my-variable-value
-            'updateTime': '2016-04-14T21:21:54.5000Z',
+            'updateTime': '2016-04-14T21:21:54.123456789Z',
```

Comment:
Could we keep a test around that doesn't have nanosecond precision in case the API changes again?

Reply:
There is one here:
[ms precision](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/runtimeconfig/tests/unit/test_variable.py#L162)

Reply:
well, I'll write another one in variable.